packages/api/test/e2e/rx-queries.spec.ts

```diff
@@ -51,4 +51,181 @@ describe.skip('Rx e2e queries', () => {
         done();
       });
   });
+
+  describe('with plain type', async () => {
+    const TOTAL_ISSUANCE = 12582912;
+    it('queries correct value', async (done) => {
```

Comment:
We are using `async` for all the rxjs tests - should not be needed since we are not using `await`?

Reply:
My mistaken.